packages/foreman/foreman/foreman.spec

```diff
@@ -585,9 +585,10 @@ Meta package to install requirements for Rackspace compute resource support.
 Summary: Foreman VMware support
 Group:  Applications/System
 # start specfile vmware Requires
-Requires: %{?scl_prefix}rubygem(fog-vsphere) >= 2.1.1
-Requires: %{?scl_prefix}rubygem(fog-vsphere) < 2.2.0
+Requires: %{?scl_prefix}rubygem(fog-vsphere) >= 2.3.0
 # end specfile vmware Requires
+Requires: %{?scl_prefix}rubygem(rbvmomi) >= 1.9.0
```

Comment:
Could you move this one line higher? That way the requirements for `bundler.d/vmware.rb` are within the `start` and `end` block. We have some automation that uses this.